docs/secrets.md

```diff
@@ -73,6 +73,10 @@ of very large secrets which would exhaust apiserver and kubelet memory.
 However, creation of many smaller secrets could also exhaust memory.  More
 comprehensive limits on memory usage due to secrets is a planned feature.
 
+Pods must be created via the API server to use secrets.  (Kubelets support
+running pods which are described in node-local files, or pulled from an
```

Comment:
I hope we will be able to change this in the future.
On Fri, May 29, 2015 at 8:42 PM CJ Cullen <notifications@github.com> wrote:

> In docs/secrets.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8979#discussion_r31375353>
> :
>
> > @@ -73,6 +73,10 @@ of very large secrets which would exhaust apiserver and kubelet memory.
> >  However, creation of many smaller secrets could also exhaust memory.  More
> >  comprehensive limits on memory usage due to secrets is a planned feature.
> >
> > +Pods must be created via the API server to use secrets.  (Kubelets support
> > +running pods which are described in node-local files, or pulled from an
>
> an URL => a URL
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8979/files#r31375353>
> .
>